overlord/snapstate/snapstate.go

```diff
@@ -221,6 +221,15 @@ type InsufficientSpaceError struct {
 	Message string
 }
 
+// ManyFlags is used in some *Many() functions to pass around flags
+// specific to actions that affect multiple snaps simultaneously.
+type ManyFlags struct {
```

Comment:
lots of fields in Flags are already very contextual, so not sure we need to introduce a new type. If we were really worried about the confusion we would likely want to introduce per operation Options structures that have a way to translate themselves into the more internal Flag structure.

So I would just add the field to Flags and pass optional *Flags to the places that don't take one yet.

The main confusion is that InstallMany doesn't respect any Flags atm, we could change that to follow what InstallPathMany does. In practice from an exernal API POV we would still not use most of them for InstallMany.

Reply:
Alright, I'll add the Transactional field to snapstate.Flags.